samples/wcfapp/README.md

```diff
@@ -0,0 +1,118 @@
+```
+
+You can also [download the repository as a zip](https://github.com/microsoft/dotnet-framework-docker/archive/master.zip).
+
+## Build and run the sample with Docker
+WCF service is supported on .NET Framework, which can run in Windows Server Core based containers. For simplicity, we disabled security in these samples.
+
+### Build a Container with IIS-hosted WCF Service
+Project `WcfServiceWebApp` is created from 'WCF Service Application' template in Visual Studio. A [Dockerfile](/Dockerfile.web) is added to the project. We use `microsoft/wcf` image as the base image, which has both HTTP and NET.TCP protocols enabled in IIS and exposes ports 80 (for HTTP) and 808 (for NET.TCP) for the container. We use WCF image with tag `4.7.2` for .NET Framework 4.7.2 in this example, but you can change it to use other tags (eg. `microsoft/wcf:4.7` with tag `4.7`) for WCF images with different versions of .NET Framework. The complete list of [WCF image tags](https://hub.docker.com/r/microsoft/wcf/tags/) can be found from Docker Hub. 
+
+Run commands below to build the container image with name `wcfservice:iis-hosted` and start an instance of it named `myservice1`. Docker parameter `-d` will run the container in background (detached mode).
+```
+cd samples
+cd wcfapp
+docker build --pull -t wcfservice:iis-hosted -f Dockerfile.web .
+docker run -d --name myservice1 wcfservice:iis-hosted
+```
+Find the IP address of the container instance. This will be used later for a WCF client to connect to the service.
+```
+docker inspect --format="{{.NetworkSettings.Networks.nat.IPAddress}}" myservice1
+172.23.70.146
+```
+### Build a Container with Self-hosted WCF Service
+Project `WcfServiceConsoleApp` is created from Windows Classic Desktop 'Console App' template in Visual Studio. We added a [Dockerfile](/Dockerfile.console) to the project. We use `microsoft/dotnet-framework` image as the base image and expose ports 80 (for HTTP) and 808 (for NET.TCP) for the container.
+
+Run commands below to build the container image with name `wcfservice:self-hosted` and start an instance of it named `myservice2`.
+```
+docker build --pull -t wcfservice:self-hosted -f Dockerfile.console .
+docker run -it --rm --name myservice2 wcfservice:self-hosted
+```
+Open another console window to find the IP address of the self-hosted WCF service (alternatively, you can run the self-hosted WCF service container in detached mode by `docker run -d --name myservice2 wcfservice:self-hosted`).
+```
+C:\wcfapp\WcfClientNetCore>docker inspect --format="{{.NetworkSettings.Networks.nat.IPAddress}}" myservice2
+172.23.69.75
+```
+### Build a Container to run WCF Client against the Service
+Now that we have WCF services running in containers. Let's run the WCF client against them. Run commands below to build the container image with name `wcfclient`.  
+```
+docker build --pull -t wcfclient -f Dockerfile.client .
+docker run -it --name myclient wcfclient
```

Comment:
Our samples should be encouraging the use of `--rm` to promote cleaning up the Docker resources when done.  I think this usage is a good candidate for using `--rm`.